### PR TITLE
API contract update for the pagination method enhancement on the users state API

### PR DESCRIPTION
**Pull request by Achintya-Chatterjee**
Date: 03 February 2024

Developer Name: @Achintya-Chatterjee 

---

## Issue Ticket Number

https://github.com/Real-Dev-Squad/website-dashboard/issues/529

### Documentation Updated?

- [x] Yes
- [ ] No

### Under Feature Flag

- [x] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

File: users/README.md [hunk 1]
```diff
@@ -216,6 +215,61 @@ Returns badges assigned to the user
...
+
+- **Params:**  
+  None
+
+- **Query Parameters:**
+  - Optional: `levelId=[string]` (Specifies the level ID)
+  - Optional: `levelName=[string]` (Specifies the level name)
+  - Optional: `levelValue=[number]` (Specifies the level value)
+  - Optional: `tagId=[string]` (Specifies the tag ID)
+  - Optional: `state=[string]` (Specifies the user state, can be repeated for multiple states)
```
Comment (prakashchoudhary07) [inline users/README.md:230]: NIT: if you can specify all states here it will be helpful

File: users/README.md [hunk 2]
```diff
@@ -216,6 +215,61 @@ Returns badges assigned to the user
...
+  - Optional: `role=[string]` (Specifies the user role, valid values are "MEMBER", "INDISCORD", "ARCHIVED")
+  - Optional: `verified=[string]` (Specifies if the user is verified)
```
Comment (prakashchoudhary07) [inline users/README.md:232]: NIT: if you can add the value of it here then, it will be helpful

**Review by sahsisunny — APPROVED** [review]

File: users/README.md (hunk 2 above)
Comment (Achintya-Chatterjee) [reply users/README.md]: Added

File: users/README.md (hunk 1 above)
Comment (Achintya-Chatterjee) [reply users/README.md]: Added

**Pull request merged**